server/config.php

```diff
@@ -10,7 +10,7 @@
 		$db_user = 'root';
 		$db_pass = 'root';
 
-		$DB_CONN = new PDO('mysql:host=127.0.0.1;dbname=vita;', $db_user, $db_pass);
+		$DB_CONN = new PDO('mysql:host=127.0.0.1:8889;dbname=vita;', $db_user, $db_pass);
```

Comment:
The default mysql port is 3306, so that's what it would probably be for production. I vote leave it without the port number and just change your local server to use 3306. That way there will be less to change from development to production.